_data/projects/dyor.yml

```diff
@@ -0,0 +1,12 @@
+name: DYOR
+desc: This is an open-source tool for doing research in crypto. You can easily look up real-time data (prices, TVLs, on-chain data) about Tokens, Exchanges, NFTs, Blockchains, etc.
+site: https://github.com/publikphigor/dyor
+tags:
+  - hacktoberfest
+  - web3
+  - cryptocurrency
+  - defi
+  - nft
+upforgrabs:
+  name: get quick github badges!
+  link: https://github.com/publikphigor/dyor/labels/get%20quick%20github%20badges!
```

Comment:
There are no issues associated with this label, which is why CI is warning about the label not being correct.

Unless there are plans to add issues to this label the project is not likely to be listed on the site after merging.

Reply:
Alright. I'd fix that.